libs/designer/src/lib/core/state/panel/panelInterfaces.ts

```diff
@@ -1,6 +1,6 @@
 import type { PanelLocation } from '@microsoft/designer-ui';
 
-export type PanelMode = 'Operation' | 'Discovery' | 'WorkflowParameters' | 'NodeSearch' | 'Error' | 'Connection';
+export type PanelMode = 'Operation' | 'Discovery' | 'WorkflowParameters' | 'NodeSearch' | 'FlowChecker' | 'Connection';
```

Comment:
Still doing validation, adding comment to block merge

Reply:
Will this cause this to be a breaking change to the Hosts?

Reply:
yes should be minor, I've called it out in the PR description. For example for PAuto to open the panel we will have to specify the new name.

Reply:
Why are we changing the underlying name then? We should be able to change the text without changing the names under,Error still makes plenty of sense.

Reply:
There are a couple components and classnames that use 'Error' which are now shared (ex. nodeErrorCard, nodeErrors, node-error-category-title, etc.).
I thought it be clearer to rename all references of 'ErrorsPanel' to FlowCheckerPanel and remaining references of 'error' to 'message'.

Not a strong preference, so if you would like to keep a mix such as keeping 'errorsPanel.tsx' as is and doing nodeErrors -> nodeMessages.tsx, node-error-category -> node-message-category I can do that